### PR TITLE
feat(angular): update highlight query

### PR DESCRIPTION
**Pull request by ninoDeme**
updated angular highlight query to support `@let` expressions

### CODE REVIEW

File: queries/angular/highlights.scm [hunk 1]
```diff
@@ -68,6 +68,9 @@
 ((control_keyword) @keyword.exception
   (#eq? @keyword.exception "error"))
 
+((control_keyword) @keyword
+  (#eq? @keyword "let"))
```
Comment (clason) [inline queries/angular/highlights.scm:72]: Wouldn't it be easier to capture `(control_keyword) @keyword` first and then override the special cases?

File: queries/angular/highlights.scm [hunk 2]
```diff
@@ -68,6 +68,9 @@
...
+
 (special_keyword) @keyword
```
Comment (clason) [inline queries/angular/highlights.scm:74]:
```suggestion
[
  (control_keyword)
  (special_keyword)
] @keyword
```
and move up

**Comment by clason** [comment]
@dlvandenberg

File: queries/angular/highlights.scm (hunk 1 above)
Comment (ninoDeme) [reply queries/angular/highlights.scm]: makes sense

**Pull request merged**